src/Shared/Resources/Strings.shared.resx

```diff
@@ -262,10 +262,26 @@
     <value>Path: {0} exceeds the OS max path limit. The fully qualified file name must be less than {1} characters.</value>
     <comment></comment>
   </data>
+  <data name="JsonParsingError" UESanitized="false" Visibility="Public">
```

Comment:
Aside for @rainersigwald afaik `UESanitized` has not been used since approx 2004 and can be removed everywhere unless you know differently

Reply:
I don't know differently. @tmeschter should we remove 'em all?

Reply:
You can remove them all. They don't affect the .xlf files in any way, and so aren't part of the localization pipeline.